libcxx/test/std/containers/container.adaptors/flat.map/flat.map.iterators/reverse_iterator.pass.cpp

```diff
@@ -28,7 +28,6 @@
 #include <iterator>
 
 #include "test_macros.h"
-#include <iostream>
```

Comment:
The `<vector>` and `<iterator>` includes above seem to be unused as well. Probably also the case in the other tests.

Reply:
This is somehow tricky. These files don't explicitly write `vector` but implicitly use `vector<T>` as default template arguments. Likewise, the reverse iterator types are `reverse_iterator` specializations, which are guaranteed to be provided in `<iterator>`.

Reply:
We have to provide these things though `<flat_map>`.

Reply:
Oh, I found that these files use `std::distance`, which is only guaranteed to be available in `<iterator>` and `<ranges>`, although transitively introduced by several headers in libc++.

I think we should either keep includes of `<iterator>` or replace these `std::distance` calls with `operator-`.

Reply:
When they're used we can keep them.